test/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/test_pixel.hpp
      DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/bin/)
 file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/test_compare.hpp
      DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/bin/)
+     file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/test_exception.hpp
```

Comment:
formatting is off (too many idents in this line), we should probably just all use https://github.com/cheshirekow/cmake_format for a consistent style

Reply:
Danke für den Hinweis. In meinem folgenden Commit müsste ich aber mit make format die Formatierung korrigiert haben

Reply:
`make format` formattiert zur zeit nur C src und header dateien keine cmake dateien

Reply:
Ok, dann benutze ich absofort nur noch cmake-format

Reply:
Danke